AutoMatrix/AutoMatrix.py

```diff
@@ -0,0 +1,837 @@
+
+import logging
+import os,sys,time,zipfile,urllib.request,shutil
+
+import vtk
+
+import slicer
+from slicer.ScriptedLoadableModule import *
+from slicer.util import VTKObservationMixin
+
+import qt
+import glob
+import numpy as np
+from qt import QFileDialog,QMessageBox
+from functools import partial
+import SimpleITK as sitk
+
```

Comment:
tqdm isn't installed by default, so importing the `Matrix_CLI` libraries fails. If it's not installed, install it via `slicer.util.pip_install`

```py
try:
    import tqdm
except ImportError:
    slicer.util.pip_install('tqdm')
    import tqdm
```

Reply:
I remove the folder Matrix_CLI in the other file I don't use tqdm so I think it's good now, thank you !